test/tap/groups/mariadb10-galera/pre-proxysql.bash

```
@@ -0,0 +1,61 @@
+
+run_admin() {
+    docker exec "${PROXY_CONTAINER}" mysql -uadmin -padmin -h127.0.0.1 -P6032 -e "$1" 2>&1 | grep -vP 'mysql: .?Warning'
+}
+
+echo ">>> Pre-proxysql: configuring Galera group (INFRA=${INFRA}, host=${MYSQL_HOST})"
+
+# Wait for ProxySQL admin to be reachable
+echo -n ">>> Pre-proxysql: waiting for ProxySQL admin..."
+for i in $(seq 1 30); do
+    if run_admin "SELECT 1" >/dev/null 2>&1; then echo " OK"; break; fi
+    if [ $i -eq 30 ]; then echo " TIMEOUT"; exit 1; fi
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)